### PR TITLE
Add clarity for framework versions supporting TLS1.3

### PR DESCRIPTION
**Pull request by blowdart**
## Summary

Trying to clear up confusion around 1.3 and framework versions

### CODE REVIEW

**Review by Rick-Anderson — APPROVED** [review]

File: docs/framework/network-programming/tls.md [hunk 1]
```diff
@@ -26,7 +26,8 @@ This document targets developers who are:
 
 We recommend that you:
 
-- Target .NET Framework 4.7 or later versions on your apps. Target .NET Framework 4.7.1 or later versions on your WCF apps.
+- For TLS 1.2 target .NET Framework 4.7 or later versions on your apps and target .NET Framework 4.7.1 or later versions on your WCF apps.
```
Comment (gewarren) [inline docs/framework/network-programming/tls.md:29]:
```suggestion
- For TLS 1.2, target .NET Framework 4.7 or later versions on your apps, and target .NET Framework 4.7.1 or later versions on your WCF apps.
```

File: docs/framework/network-programming/tls.md [hunk 2]
```diff
@@ -26,7 +26,8 @@ This document targets developers who are:
...
+- For TLS 1.3 target .NET Framework 4.8 or later.
```
Comment (gewarren) [inline docs/framework/network-programming/tls.md:30]:
```suggestion
- For TLS 1.3, target .NET Framework 4.8 or later.
```

File: docs/framework/network-programming/tls.md [hunk 3]
```diff
@@ -26,7 +26,8 @@ This document targets developers who are:
 
 We recommend that you:
```
Comment (gewarren) [inline docs/framework/network-programming/tls.md:27]:
```suggestion
Consider the following recommendations:
```

**Review by gewarren — APPROVED** [review]
I changed the opening text since it didn't flow as "We recommend that you for TLS 1.2 target .NET Framework...".

**Comment by gewarren** [comment]
Thanks @blowdart

**Pull request merged**